src/app/forms/draft-court-order-conviction/draft-court-order-conviction.pdf.component.js

```diff
@@ -87,23 +87,10 @@ export default class DraftCourtOrderConviction_Pdf extends React.Component {
           <PositionedString dataKey="case.judgeName" left="54.1%" top="55.1%" />
 
           <PositionedCheckmark
-            dataKey="case.resolutionType"
-            left="12.1%"
-            top="68.8%"
-            shouldShow={data.case.resolutionType === "hearing"}
-          />
-
-          <PositionedCheckmark
-            dataKey="case.resolutionType"
+            debugKey="resolutionType"
```

Comment:
Should use `datakey`.

- `dataKey`: when PositionedCheckmark should render data itself
- `debugKey`: when PositionedCheckmark shouldn't render the data, but the `props.children` will include the data.

Maybe we should update all the components to enforce this so that we don't have to worry about the distinction because things would break if done incorrectly.

Reply:
Why did you remove `case.` from this?

```suggestion
            dataKey="case.resolutionType"
```

Reply:
There is no corresponding ```dataKey``` on the web component.

Reply:
Ah okay